packages/bot-web-ui/src/components/dashboard/dashboard-component/intro-card.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+type TIntroCard = {
+    label: string;
+    content: string[];
+    index: number;
+    sidebar_item: string[] | any;
```

Comment:
```suggestion
    sidebar_items: string[];
```
any reason why this is typed as `any`? Best to name it with a plural since you are expecting an array of string

Reply:
done ✅